CMakeLists.txt

```diff
@@ -51,81 +135,60 @@ install(
 	TARGETS DDC
 	EXPORT DDCTargets)
 
+	
 ## The PDI wrapper
-if(DDC_BUILD_PDI_WRAPPER)
-	if(NOT TARGET PDI::PDI_C)
-		find_package(PDI REQUIRED COMPONENTS C)
-	endif()
 
-	add_library(PDI INTERFACE)
-	target_compile_features(PDI INTERFACE cxx_std_17)
-	target_include_directories(PDI
+if(DDC_BUILD_PDI_WRAPPER)
+	add_library(PDI_wrapper INTERFACE)
```

Comment:
Projects will use `DDC::PDI_wrapper` right ?

Should we find a naming convention ?

Reply:
I have no issue with `DDC::PDI`, however, just `PDI` can clash with a PDI library if including both

Reply:
Really ? Is there a `PDI` target that is exported ?

Reply:
The one I renamed `PDI_wrapper` used to be named `PDI`, no?

Reply:
I mean in the PDI library. Clearly if a project is embedding both PDI and DDC this will break. That is a limitation of embedding libraries as submodules I guess.

Reply:
Yes we should recommend to install libraries instead of embedding the world :)

Reply:
yes, use spack!

Reply:
I thought there was an agreement on keeping DDC::PDI then ;)

Reply:
Yes, but then I realized that when you find package, the target name will be `DDC::PDI_wrapper` anyway and i throught that was what you were telling me first